### PR TITLE
Load minified JS in cross browser tests

### PR DESCRIPTION
**Pull request by theninj4**
This PR includes the minified JavaScript assets within the Karma test runner, to validate all the browsers are happy with their encoding.

I've had to create a lesser minified version of barometer to get things working in IE7 and 8.

### CODE REVIEW

**Comment by theninj4** [comment]
IE7 and IE8 don't like the crushed JS:
```
Chrome 52.0.2743 (Windows 10 0.0.0): Executed 17 of 17 SUCCESS (0.147 secs / 0.045 secs)
Firefox 47.0.0 (Windows 10 0.0.0): Executed 17 of 17 SUCCESS (0.229 secs / 0.011 secs)
Edge 13.10586.0 (Windows 10 0.0.0): Executed 17 of 17 SUCCESS (0.205 secs / 0.017 secs)
Safari 9.0.0 (Mac OS X 10.11.0): Executed 17 of 17 SUCCESS (0.176 secs / 0.011 secs)
IE 11.0.0 (Windows 10 0.0.0): Executed 17 of 17 SUCCESS (0.186 secs / 0.006 secs)
IE 9.0.0 (Windows 7 0.0.0): Executed 17 of 17 SUCCESS (0.222 secs / 0.003 secs)
Chrome Mobile 39.0.0 (Android 5.1.0): Executed 17 of 17 SUCCESS (0.186 secs / 0.048 secs)
Mobile Safari 9.0.0 (iOS 9.2.0): Executed 17 of 17 SUCCESS (0.171 secs / 0.016 secs)
Mobile Safari 7.0.0 (iOS 7.1.0): Executed 17 of 17 SUCCESS (0.166 secs / 0.006 secs)
TOTAL: 153 SUCCESS
```
```
IE 8.0.0 (Windows XP 0.0.0) ERROR
  Invalid character
  at dist/barometer.min.js:2
IE 7.0.0 (Windows XP 0.0.0) ERROR
  Invalid character
  at http://localhost:9876/context.html:3
IE 7.0.0 (Windows XP 0.0.0): Executed 0 of 0 ERROR (1.636 secs / 0 secs)
IE 8.0.0 (Windows XP 0.0.0): Executed 0 of 0 ERROR (0.928 secs / 0 secs)
```
We may end up providing an additional non-crushed build.

File: dist/barometer-test.js [hunk 1]
```diff
@@ -139,7 +139,9 @@ pageLoadStats._normaliseStats = function (stats) {
   var normalisedStats = { }
```
Comment (shackpank) [inline dist/barometer-test.js]: Should `barometer-ie.min.js` be committed to the repo, and this file not be?

Comment (theninj4) [inline dist/barometer-test.js]: Yes, I think you're right :+1:

**Comment by hxgeorge** [comment]
👍

**Pull request merged**